orhun/git-cliff
# fix(changelog): fix missing commit fields in context (#837)

## Description

Add new fields `raw_message` in `Commit` to avoid lossy message conversion.

Fix #837 

## Motivation and Context
I implemented according to https://github.com/orhun/git-cliff/issues/837#issuecomment-2404818161



## How Has This Been Tested?




Add 2 test fixtures
- Create changelog with all fields from conventional commits
- Run `git cliff --context | git cliff --from-context` should yield identical result with `git cliff`

## Screenshots / Logs (if applicable)

## Types of Changes



- [ ] Documentation (no code change)
- [ ] Refactor (refactoring production code)

## Checklist:




- [x] I have formatted the code with [rustfmt](https://github.com/rust-lang/rustfmt).
- [x] I checked the lints with [clippy](https://github.com/rust-lang/rust-clippy).
